lib/modules/datasource/docker/index.spec.ts

```diff
@@ -1193,32 +1193,45 @@ describe('modules/datasource/docker/index', () => {
       await expect(getPkgReleases(config)).rejects.toThrow(EXTERNAL_HOST_ERROR);
     });
 
-    it.each([[true], [false]])(
-      'jfrog artifactory - retry tags for official images by injecting `/library` after repository and before image, abortOnError=%p',
-      async (abortOnError) => {
-        hostRules.find.mockReturnValue({ abortOnError });
-        const tags = ['18.0.0'];
-        httpMock
-          .scope('https://org.jfrog.io/v2')
-          .get('/virtual-mirror/node/tags/list?n=10000')
-          .reply(200, '', {})
-          .get('/virtual-mirror/node/tags/list?n=10000')
-          .reply(404, '', { 'x-jfrog-version': 'Artifactory/7.42.2 74202900' })
-          .get('/virtual-mirror/library/node/tags/list?n=10000')
-          .reply(200, '', {})
-          .get('/virtual-mirror/library/node/tags/list?n=10000')
-          .reply(200, { tags }, {})
-          .get('/')
-          .reply(200, '', {})
-          .get('/virtual-mirror/node/manifests/18.0.0')
-          .reply(200, '', {});
-        const res = await getPkgReleases({
-          datasource: DockerDatasource.id,
-          depName: 'org.jfrog.io/virtual-mirror/node',
-        });
-        expect(res?.releases).toHaveLength(1);
-      }
-    );
+    it('jfrog artifactory - retry tags for official images by injecting `/library` after repository and before image', async () => {
+      const tags1: string[] = [...Array(10000)].map((_, i) => `${i}.0.0`);
```

Comment:
i think we've a range util function for this

Reply:
I see what you are referencing. I've made an update.